Mathlib/Analysis/InnerProductSpace/Adjoint.lean

```diff
@@ -40,6 +40,7 @@ adjoint
 
 -/
 
+local macro_rules | `($x ^ $y)   => `(HPow.hPow $x $y) -- Porting note: See issue #2220
```

Comment:
This isn't #2220; what does this refer to?

Reply:
https://github.com/leanprover/lean4/issues/2220

Reply:
```suggestion
local macro_rules | `($x ^ $y)   => `(HPow.hPow $x $y) -- Porting note: See issue lean4#2220
```